src/renderer/hooks/useRunePrice.types.ts

```diff
@@ -0,0 +1,6 @@
+import * as RD from '@devexperts/remote-data-ts'
+import * as O from 'fp-ts/lib/Option'
+
+import { AssetWithAmount } from '../types/asgardex'
+
+export type RunePriceRD = RD.RemoteData<Error, O.Option<AssetWithAmount>>
```

Comment:
not sure that i understand why we need option inside ?

Reply:
Price might not be available in case there is no price pool or there is no selected price asset.

Reply:
yep. but is not `RD.initial` enough in this case ?
- `initial` : no data provided
- `pending`: loading data
- `error`: error data fetching
- `success`: show data
?

Reply:
^ solved by @thatStrangeGuyThorchain :eye:  a1c50638f119bdb30c5ff6f81adef3cfd9c5a5f3